assets/js/team.tsx

```diff
@@ -67,42 +71,66 @@ function AlumRoster({ members }: { members: Alum[] }) {
   );
 }
 
-export function loadTeam() {
-  const currentUser = loggedInUser();
+function Rosters() {
+  const nav = document.getElementById("sidenav-wrapper") as HTMLElement;
+  const [currentUser] = useCurrentUser();
 
-  if (currentUser) {
-    const teamContainer = document.getElementById(
-      "team-container"
-    ) as HTMLElement;
-    const alumniContainer = document.getElementById(
-      "alumni-container"
-    ) as HTMLElement;
+  const emptyData = { teamMembers: [], alumni: [] };
+  const { data } = useQuery("team.yml", () => {
+    if (!currentUser) {
+      return emptyData;
+    }
 
-    fetchGitHubFile({
+    return fetchGitHubFile({
       token: currentUser.token,
       repo: "18f/identity-private",
       path: "team/team.yml",
     }).then((file) => {
       if (Array.isArray(file)) {
-        return;
+        return emptyData;
       }
       const { team_members: teamMembers, alumni } = loadYAML(
         atob(file.content)
       ) as TeamRoster;
 
-      render(<TeamMemberRoster members={teamMembers} />, teamContainer);
-      render(<AlumRoster members={alumni} />, alumniContainer);
+      return { teamMembers, alumni };
     });
-  } else {
-    const errorContainer = document.getElementById(
-      "error-container"
-    ) as HTMLElement;
+  });
+
+  return (
+    <>
+      <h3>
+        Current Team Members <AnchorLink slug="current-team-members" />
+      </h3>
+      {data?.teamMembers?.length && (
```

Comment:
This is prone to super common issue where left-hand falsiness of `0` will be rendered _literally_, outputting `0` instead of nothing ([example](https://codepen.io/aduth/pen/dWwzrL)).

```suggestion
      {!!data?.teamMembers?.length && (
```

Reply:
oh good point